frontend/src/metabase/query_builder/components/notebook/steps/JoinStep/JoinStep.unit.spec.tsx

```diff
@@ -278,6 +284,39 @@ describe("Notebook Editor > Join Step", () => {
     ).not.toBeInTheDocument();
   });
 
+  it("questions from another database should not appear in recents", async () => {
```

Comment:
Please always include the tag for repros :pray: 

```suggestion
  it("questions from another database should not appear in recents (metabase#44974)", async () => {
```